distributed/worker.py

```diff
@@ -999,7 +999,7 @@ def func(data):
                 cache_loads.data.clear()
             except Exception as e:
                 logger.exception(e)
-                return {"status": "error", "exception": to_serialize(e)}
+                return {"status": "error", "exception": e}
```

Comment:
Exception handling is now handled by the worker plugin system, so this no longer should be serialized.

Reply:
Nice! I think we can simplify a little further and just raise the exception here directly

```suggestion
                raise e
```